packages/e2e-tests/specs/editor/plugins/__snapshots__/cpt-locking.test.js.snap

```diff
@@ -67,5 +83,9 @@ exports[`cpt locking template_lock insert should allow blocks to be moved 1`] =
 
 <!-- wp:quote -->
 <blockquote class=\\"wp-block-quote\\"><p></p></blockquote>
-<!-- /wp:quote -->"
+<!-- /wp:quote -->
+
+<!-- wp:columns -->
+<div class=\\"wp-block-columns\\"></div>
+<!-- /wp:columns -->"
```

Comment:
Not sure I understand why the snapshots change since you add a new test?

Reply:
I added a columns block to the default template so I can test a CPT locked all with a block without locking. The existing snapshots need to be updated to contain the added columns block.